hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/logaggregation/filecontroller/LogAggregationFileControllerFactory.java

```diff
@@ -66,76 +65,64 @@ public LogAggregationFileControllerFactory(Configuration conf) {
     Collection<String> fileControllers = conf.getStringCollection(
         YarnConfiguration.LOG_AGGREGATION_FILE_FORMATS);
     List<String> controllerClassName = new ArrayList<>();
-
     Map<String, String> controllerChecker = new HashMap<>();
 
-    for (String fileController : fileControllers) {
+    for (String controllerName : fileControllers) {
       Preconditions.checkArgument(validateAggregatedFileControllerName(
```

Comment:
I think the validateAggregatedFileControllerName is only used here. Can we move this big validation message and argument checking in that method instead? It would be much more readable.

Reply:
Sure, good catch. Moved it.